.github/ISSUE_TEMPLATE/patch_release.md

```diff
@@ -30,7 +30,8 @@ examples of each step, assuming vX.Y.Z is being cut.
   - [ ] `xp/getting-started-with-gcp`
 - [ ] Confirm the full set of patch versions that will be released and promote them from lowest to highest, so the **highest** version is the **last** to be promoted (e.g. `v1.12.2` should be promoted after `v1.11.3`), in order to avoid the promote workflow overwriting the latest patch release.
 - [ ] Run the [Promote workflow][promote-workflow] with channel `stable` on the `release-X.Y` branch and verified that the tagged build version exists on the [releases.crossplane.io] `stable` channel at `stable/vX.Y.Z/...`.
-- [ ] Published a [new release] for the tagged version, with the same name as the version and descriptive release notes, taking care of generating the changes list selecting as "Previous tag" `vX.Y.<Z-1>`, so the previous patch release for the same minor. Before publishing the release notes, set them as Draft and ask the rest of the team to double check them.
+- [ ] Published a [new release] for the tagged version, with the same name as the version and descriptive release notes filling in the missing fields. Start from the [DRAFT_RELEASE.md][draft-release] file on the `release-X.Y` branch, filling in all the missing sections, taking care of generating the changes list selecting as "Previous tag" `vX.Y.<Z-1>`, so the previous patch release for the same minor. Before publishing the release notes, set them as Draft and ask the rest of the team to double check them.
+- [ ] Created and merged a PR cleaning up the [DRAFT_RELEASE.md][draft-release] file on the `release-X.Y` branch, preparing it for the next patch version, `vX.Y.Z+1`.
```

Comment:
Not really sure if it's worth the overhead for patch releases. 
We usually have a small number of commits between patches (and no intentional breaking changes 🤞).

Reply:
Sounds good, should we wipe the file on release branches then?

Reply:
Agreed!